src/main/java/edu/cornell/kfs/paymentworks/service/PaymentWorksUploadSupplierService.java

```diff
@@ -24,9 +24,18 @@
 	String createSupplierUploadFile(List<PaymentWorksSupplierUploadDTO> paymentWorksSupplierUploadList,String directoryPath);
 	
 	/**
-	 * Deletes the file passed in.
-	 * @param fileName
+	 * Uploads approved suppkiers to PaymentWorks
 	 */
-	void deleteSupplierUploadFile(String fileName);
```

Comment:
Curious.  How will the removal of bad or unneeded files that were uploaded be dealt with without this method?